Chart.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 apiVersion: v2
 name: nifi
-version: 1.0.6
+version: 1.0.7
```

Comment:
Due to the vast amount of changes, this should probably be 1.1.x or maybe even 2.0.0

Reply:
Yes - I've tried really hard to avoid breaking backwards compatibility (to include writing tests for [e.g.] LDAP), so don't think it needs a major version bump.